front/src/modules/trainschedule/components/DriverTrainScheduleV2/DriverTrainScheduleHeaderV2.tsx

```diff
@@ -46,7 +46,7 @@ const DriverTrainScheduleHeaderV2 = ({
     <>
       <div className="d-flex align-items-center">
         <h1 className="text-blue flex-grow-1">{train.train_name}</h1>
-        {isEco(simulatedTrain) && (
+        {isEco(simulatedTrain) || (train.schedule && train.schedule?.length > 1) ? (
```

Comment:
This doesn't check if there is a scheduled point.

You can have a non-empty `train.schedule` array but without a scheduled point:

```json
{
    // ...
    "schedule": [
      {
        "at": "id2544",
        "arrival": null, // <-- If arrival field is set to anything but `null` it's a scheduled point
        "stop_for": "P0D",
        "on_stop_signal": false,
        "locked": false
      }
    ]
}
```